Explorer/Assets/DCL/WebRequests/RequestEnvelope.cs

```diff
@@ -68,7 +68,8 @@ public TWebRequest InitializedWebRequest(IWeb3IdentityCache web3IdentityCache)
 
         public void Dispose()
         {
-            headersInfo?.Dispose();
```

Comment:
But what's the difference? Which problem you solved this way?

Reply:
That headersInfo might be null, if struct has been created directly

Reply:
public readonly IReadOnlyList<WebRequestHeader> Value => values as IReadOnlyList<WebRequestHeader> ?? Array.Empty<WebRequestHeader>();